build.info

```diff
@@ -3,7 +3,7 @@
 SUBDIRS=crypto ssl apps test util tools fuzz engines providers
 
 LIBS=libcrypto libssl
-INCLUDE[libcrypto]=. crypto/include include
+INCLUDE[libcrypto]=. crypto/include include providers/common/include
```

Comment:
I was unsure if I should comment on this...  but I'm wondering if this shouldn't be kept with the default provider build.info.  My reasoning is that if we decide to make it possible to make the default provider a module as well some time in the future, we shouldn't need to make a default provider conditional here, or to chase around the build.info files to find this particular include setting.

Reply:
This was actually implemented by #8700, but I've added a new commit in this PR to change it as you suggest.